x-pack/test/fleet_api_integration/apis/agent_policy/agent_policy.ts

```diff
@@ -297,6 +298,83 @@ export default function (providerContext: FtrProviderContext) {
         });
       });
 
+      it('should increment package policy copy names', async () => {
```

Comment:
logic like this can be easily covered with unit test I think

Reply:
👍  Just added a unit test